docs/installation/expansion.rst

```diff
@@ -237,11 +237,25 @@ Open a terminal in the Salt Master container using
 #. Check that SSH access from the Salt Master to the new node is properly
    configured (see :ref:`Bootstrap SSH Provisioning`).
 
+   .. note::
+
+     You need to use `salt-ssh` as raw ssh since `salt-ssh` need Python3
+     installed to work properly (that will be installed by the following
+     salt orchestrate).
```

Comment:
Not sure this note is relevant to our readers, but if it is, we should reformulate it a bit:

```
Salt SSH requires Python 3 to be installed on the remote host to run Salt functions.
It will be installed automatically when deploying the node, though you can send raw
shell commands before (using ``--raw-shell``) if needed.
```

(side-note: literal text needs double backticks **``** in RST (or the `:literal:` role), otherwise it's interpreted differently and we cannot guarantee how it will be displayed - see [the docs](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#interpreted-text))

Reply:
Right, 
I think it's relevant because they cannot run directly `test.ping` as it was documented before, I will change it as you suggest